swahili/swahili.md

```diff
@@ -1 +1,3 @@
 Mambo ndiyo hayo basi. Swahili lipo wavutini. Jiskia huru kuongezea stori...
+
+Na basi ndipo ukakata shauri (kutembelea jw nukta org)[tembelea-jw-nukta-org/tembelea-jw-nukta-org.md].
```

Comment:
Did you mean for this to be a link?  If so, your syntax is backwards.  The text should be in square brackets and the link in parentheses rather than vice versa.  You can view the page on GitHub on your fork to check if the link is displaying correctly.